workflow_creation/advanced_branching_looping.md

```diff
@@ -1,2 +1,109 @@
-# Advanded workflow creation: Branching and looping
-to be written
\ No newline at end of file
+# Advanced workflow creation: Branching and looping
+
+To create more sophisticated workflows, the workflow editor offers branching and loops:
+
+Need to perform a case distinction and execute different services based on the result? Make use of the [branching feature](#branching).
+
+Want to execute a service multiple times? There is no need to manually call it multiple times, make use of the [loops](#loops).
+
+
+### Condition scheme
+
+Both features support a simple condition scheme, denoted in xml, which enables integer comparisons:
+
+```
+<condition>COMMAND(VARIABLE, VALUE)</condition>
+```
+
+The COMMAND may one of the following: `greaterThan`, `greaterOrEqual`, `equalTo`, `lessThan`, `lessOrEqual`.
+The condition will compare the value of the VARIABLE with VALUE based on the COMMAND.
+
```

Comment:
Suggestion: Add a note that any of the workflow and service parameters currently "active" can be used as variables.

Question: Is that correct? ;-)